### DISCUSSION TITLE
How to use a previously trained model for adding new trainig samples to it

### DISCUSSION THREAD

**Discussion by dghadagh** [mljar/mljar-supervised#609] [General]
How can I use a previously trained model and add some limited count of new trainig samples to it? In other words, I want to ADD just a few new samples, without repeating the whole learning  process with the initial big dataset.

**Comment by pplonski** [comment]
Hi @dghadagh, there is no such feature. You need to trsin new model from scratch.

I think there is no such feature in sklearn models or xgboost.

**Reply by dghadagh** [reply]
Hi and thank you so much dear @pplonski
